src/libstd/collections/hash/map.rs

```diff
@@ -1577,6 +1605,13 @@ impl<K, V> ExactSizeIterator for IntoIter<K, V> {
 #[unstable(feature = "fused", issue = "35602")]
 impl<K, V> FusedIterator for IntoIter<K, V> {}
 
+#[stable(feature = "std-debug", since = "")]
+impl<K, V> fmt::Debug for IntoIter<K, V> {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
```

Comment:
Even the IntoIter might want to show the elements, even if it can not iterate itself, it can maybe borrow itself as a by-reference iterator (now or in the future). Which means `K, V` need to impl debug.